scripts/beekeeper.pl

```diff
@@ -422,13 +422,29 @@ sub generate_worker_cmd {
 
     my $worker_cmd = 'runWorker.pl';
 
-    foreach my $worker_option ('url', 'reg_conf', 'reg_type', 'reg_alias', 'nosqlvc', 'job_limit', 'life_span', 'retry_throwing_jobs', 'can_respecialize',
+    foreach my $worker_option ('url', 'reg_conf', 'reg_type', 'reg_alias', 'job_limit', 'life_span',
                                'worker_delay_startup_seconds', 'worker_crash_on_startup_prob', 'hive_log_dir', 'debug') {
         if(defined(my $value = $self->{$worker_option})) {
             $worker_cmd .= " -${worker_option} $value";
         }
     }
 
+    foreach my $worker_flag ('retry_throwing_jobs', 'can_respecialize') {
+        if(defined(my $value = $self->{$worker_flag})) {
+            if ($value == 0) {
+                $worker_cmd .= " -no${worker_flag}";
+            } else {
+                $worker_cmd .= " -${worker_flag}";
+            }
+        }
+    }
+
+    # nosqlvc is a special case for compatibility with modules where it is a propigated option
+    if ((defined($self->{nosqlvc})) &&
+        ($self->{nosqlvc} == 1)) {
+        $worker_cmd .= " -nosqlvc";
+    }
+
```

Comment:
I think `nosqlvc` could be covered by the _foreach_ loop just above as it is either undefined or set to 1